paywall/src/utils/validators.js

```diff
@@ -344,3 +343,66 @@ export const isValidBalance = balance => {
     )
   }, true)
 }
+
+const allowedInputTypes = ['text', 'date', 'color', 'email', 'url']
```

Comment:
Would appreciate comment on which fields we should include. I believe these are the only fields that don't require extra data (`select`s would need a series of options, for example)